
def log_error(e):
    print(e)
